src/client/src/rt-styleguide/components/Button.tsx

```diff
@@ -208,7 +215,8 @@ const StyledButton: any = styled(StyledButtonBase)<ButtonStyleProps>`
   }
 
   ${({ size }) =>
-    (size || size > 1) &&
+    size &&
```

Comment:
not sure why it had logical OR before, it seems to need logical AND - then types are correct and seems that logic too